### DISCUSSION TITLE
A giscus logo idea

### DISCUSSION THREAD

**Discussion by Andre601** [giscus/giscus#364] [Show and tell]
The current... logo(?) is not really something of interest. It's just the `:gem:` (💎) emoji in a particular style I suppose, which is fine, but I feel like it doesn't really give an impression of what giscus is used for (What it offers).

So, since I was bored, as usual, did I try to make a sort of logo for giscus. For that did I use two of GitHub's Octicons ([browser](https://primer.style/octicons/browser-24) and [comment-discussion](https://primer.style/octicons/comment-discussion-24)), tweaked them a bit and gave some basic colours.

The result is this one here:
![giscus logo](https://user-images.githubusercontent.com/11576465/150561157-448a806b-1320-45d5-8b6f-e41e09d14ed0.png)

Of course, there is no need to use this image. I just made it for fun and wanted to share it. But feel free to use it, if you like. I won't mind at all.

**Comment by laymonage** [comment]
> The current... logo(?) is not really something of interest. It's just the :gem: (gem) emoji in a particular style I suppose, which is fine, but I feel like it doesn't really give an impression of what giscus is used for (What it offers).

Yeah, about that... :stuck_out_tongue:

This was historically explained in #45.

I do understand your concern. I'll take this into consideration and maybe use your logo idea as inspiration. Thanks for raising this :slightly_smiling_face:
